solr/core/src/test/org/apache/solr/security/BasicAuthIntegrationTest.java

```diff
@@ -232,7 +234,7 @@ public void testBasicAuth() throws Exception {
         HttpSolrClient.RemoteSolrException e = expectThrows(HttpSolrClient.RemoteSolrException.class, () -> {
           new UpdateRequest().deleteByQuery("*:*").process(aNewClient, COLLECTION);
         });
-        assertTrue(e.getMessage().contains("Unauthorized request"));
+        assertTrue(e.getMessage(), e.getMessage().contains("Authentication failed"));
```

Comment:
Earlier both 401 and 403 responses would print the text "Unauthorized request" due to the fall-through. After this fix we also changed the text for 401 response, making this test fail. Don't know why Authorization plugin returns 401 though, the password is correct..